core/src/main/java/nl/nn/adapterframework/filesystem/LocalFileSystem.java

```diff
@@ -114,8 +114,8 @@ public OutputStream appendFile(Path f) throws IOException {
 	}
 
 	@Override
-	public InputStream readFile(Path f) throws IOException {
-		return Files.newInputStream(f);
+	public Message readFile(Path f) throws IOException {
+		return new Message(Files.newInputStream(f));
```

Comment:
Ik denk dat het mooier is als je of
- `Path` toevoegt aan `Message`
- Path` vertaald naar een `File` en die meegeeft aan `Message`